WRITING-DOCUMENTATION.md

```diff
@@ -3,86 +3,73 @@
 After starting the site locally, navigate to `http://localhost:8080/docs/`. This is where you can view your work 
 as you write your documentation.
 
-## 1. Creating a new version
-
-1. Copy the most recent release folder and rename it accordingly by its version -- each version folder is located in 
-[`/docs/docs/](/docs/docs/)
-for structuring version folders properly)
-1. Make your edits in the new release folder you've created -- you can run `yarn docs:dev` to view your changes locally
-2. Add your new release number to the `releases.json` file located in `.vuepress/public` -- please follow the number order and put the latest at the end.
-
-The website will automatically build the release list from the `releases.json` file. This is what is used for things 
-like the Install page and the version selector that appears at the top of the sidebar on the Documentation page. 
-This file will also automatically create the routes needed for each new release, which is super important.
-
-The Install and Documentation pages will also always redirect to the documentation and installation methods for the 
-latest release to ensure that our users are always looking at the latest docs at any time.
-
-## 2. Setting up the initial sidebar for new versions
-
-Because the automatic sidebar functionality of VuePress is not quite perfect, we are using a combination of its 
-automatic features, and some manual control. Within the [`sidebar-nav.js`](/docs/.vuepress/site-config/sidebar-nav.js) file, you'll see this:
-
-``` js
-// /docs/.vuepress/site-config/sidebar-nav.js
-module.exports = {
-  "/docs/0.1.0/": [
-    "",
-    "getting-started/",
-    "documentation/",
-    "tutorials/",
-    "installation/",
-    "community/"
-  ],
-  "/docs/0.2.0/": [
-    "",
-    "getting-started/",
-    "documentation/",
-    "tutorials/",
-    "installation/",
-    "community/"
-  ]
-}
-```
-This is the primary template for each version's sidebar navigation on the documentation pages.
+## Versions
 
-Each new version needs its own navigation block with the primary links below it as seen above. The blank `""` 
-below `/docs/0.2.0/` will automatically generate a nav item in the VuePress sidebar to the main page for that 
-version. The named sub-pages below each version are controlled by their markdown files that share the same name.
+We do trunk based development (master is the `trunk` branch).
 
-### Example:
+There's a folder in [docs/docs](docs/docs) for each minor version of Kuma. 
+There is a special folder for the future version of Kuma which is called [next](docs/docs/next).
```

Comment:
is it worth to clarify that the future version is `minor` version? Patch release is technically also a future version

Reply:
Will do